homeassistant/components/easee/const.py

```diff
@@ -0,0 +1,258 @@
+"""Easee Charger constants."""
+DOMAIN = "easee"
+MEASURED_CONSUMPTION_DAYS = "measured_consumption_days"
+CONF_MONITORED_SITES = "monitored_sites"
+CUSTOM_UNITS = "custom_units"
+PLATFORMS = ["sensor"]
+SCAN_INTERVAL_SECONDS = 60
+LISTENER_FN_CLOSE = "update_listener_close_fn"
+MEASURED_CONSUMPTION_OPTIONS = {
+    "1": "1",
+    "7": "7",
+    "14": "14",
+    "30": "30",
+    "365": "365",
+}
+CUSTOM_UNITS_OPTIONS = {
+    "kW": "Power kW to W",
+    "kWh": "Energy kWh to Wh",
+}
+CUSTOM_UNITS_TABLE = {
+    "kW": "W",
+    "kWh": "Wh",
+}
```

Comment:
Can you use these consts here:
POWER_WATT (= "W")
POWER_KILO_WATT (= f"k{POWER_WATT}")
ENERGY_WATT_HOUR (= f"{POWER_WATT}h")
ENERGY_KILO_WATT_HOUR (= f"k{ENERGY_WATT_HOUR}")